src/main/java/javax/validation/Configuration.java

```diff
@@ -254,6 +269,17 @@
 	ClockProvider getDefaultClockProvider();
 
 	/**
+	 * Returns the default {@link ValueExtractor} implementations as per the
+	 * specification.
+	 *
+	 * @return the default {@code ValueExtractor} implementations compliant
+	 * with the specification
+	 *
+	 * @since 2.0
+	 */
+	Set<ValueExtractor<?>> getDefaultValueExtractors();
```

Comment:
I'd leave it out and have the providers experiment with more of a Map<type,type_param,Extractor> API to see if that meta info is useful